packages/cli/src/Generate.ts

```diff
@@ -214,9 +212,9 @@ Please run \`prisma generate\` manually.`
         const generator = prismaClientJSGenerator.options?.generator
         const isDeno = generator?.previewFeatures.includes('deno') && !!globalThis.Deno
         if (isDeno && !generator?.isCustomOutput) {
-          throw new Error(`Can't find output dir for generator ${chalk.bold(
-            generator?.name,
-          )} with provider ${chalk.bold(generator?.provider.value)}.
+          throw new Error(`Can't find output dir for generator ${bold(generator!.name)} with provider ${bold(
```

Comment:
note to other reviewers if anyone else stumbles here too: `isDeno` implies `generator` is not nullish, so `!` is safe here

Reply:
Can't this be turned into a type guard that uses `isDeno` to avoid `!`?